admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js

```diff
@@ -32,19 +38,58 @@ export default class CurrencyForm {
   constructor(currencyFormMap) {
     this.map = currencyFormMap;
     this.$currencyForm = $(this.map.currencyForm);
-    this.getCLDRDataUrl = this.$currencyForm.data('get-cldr-data');
+    this.$currencyFormFooter = $(this.map.currencyFormFooter);
+    this.apiReferenceUrl = this.$currencyForm.data('reference-url');
+    this.languages = this.$currencyForm.data('languages');
+    this.translations = this.$currencyForm.data('translations');
     this.$currencySelector = $(this.map.currencySelector);
     this.$isUnofficialCheckbox = $(this.map.isUnofficialCheckbox);
     this.$isoCodeInput = $(this.map.isoCodeInput);
     this.$exchangeRateInput = $(this.map.exchangeRateInput);
     this.$precisionInput = $(this.map.precisionInput);
     this.$resetDefaultSettingsButton = $(this.map.resetDefaultSettingsInput);
     this.$loadingDataModal = $(this.map.loadingDataModal);
+    this.currencyFormatterId = this.map.currencyFormatter.replace('#', '');
   }
 
   init() {
     this._initListeners();
     this._initFields();
+    this._initStore();
+    this._initCurrencyFormatter();
+  }
+
+  _initStore() {
+    this.store = new CurrencyStore(this.apiReferenceUrl);
+    this.store.subscribe((mutation, state) => {
+      if (types.CUSTOMIZE_CURRENCY_FORMAT === mutation.type ||
+        types.RESET_LANGUAGE === mutation.type) {
+        this._fillCurrencyCustomData(state.currencyData);
+      }
+    });
+    this.store.dispatch('setLanguages', this.languages);
+    this.store.dispatch('setTranslations', this.translations);
+    this.store.dispatch('setCurrencyData', this._getCurrencyDataFromForm());
+  }
+
+  _initCurrencyFormatter() {
+    // Customizer only present when languages data are present (in edition usually)
+    if (!this.languages.length) {
+      return;
+    }
+
+    const currencyIsoCode = this.$isoCodeInput.val();
+    $(`<div class="card-block row" id="${this.currencyFormatterId}"></div>`).insertBefore(this.$currencyFormFooter);
+    this.currencyFormatter = new Vue({
+        el: this.map.currencyFormatter,
+        store: this.store,
+        components: {CurrencyFormatter},
+        template: `
+      <div id="${this.currencyFormatterId}" class="card-block row">
```

Comment:
same here, this template must be splitted in a view file and in a twig file

Reply:
As you said, you're able to add this block in your twig file
```
<div id="currency_formatter" class="card-block row"></div>
```
and have only the components in your `template` at least something editing the BO can place  this component everywhere he wants